ci/e2e-tests.sh

```diff
@@ -655,7 +655,7 @@ ssh -i "$PWD/vm-ssh-key" "aziot@$vm_public_ip" "
 
     module_identity=\"\$(
         curl --unix-socket '/run/aziot/identityd.sock' \\
-            'http://foo/identities/modules/testmodule?api-version=2020-09-01&type=aziot'
+            'http://foo/identities/identity?api-version=2020-09-01'
```

Comment:
Fix the doc comment in `iothub-get-twin.sh` too please (the example line 8).

Reply:
Done